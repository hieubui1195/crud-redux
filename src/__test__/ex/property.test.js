it('will fail every time', () => {
    const user = {
        createdAt: new Date(),
        id: Math.floor(Math.random() * 20),
        name: 'Bui Hi',
    };
    
    expect(user).toMatchSnapshot();
});

it('will check the matchers and pass', () => {
    const user = {
        createdAt: new Date(),
        id: Math.floor(Math.random() * 20),
        name: 'Bui Hi',
    };

    expect(user).toMatchSnapshot({
        createdAt: expect.any(Date),
        id: expect.any(Number),
    });
});
