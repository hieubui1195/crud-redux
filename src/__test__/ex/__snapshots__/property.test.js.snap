// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will check the matchers and pass 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "Bui Hi",
}
`;

exports[`will fail every time 1`] = `
Object {
  "createdAt": 2018-08-06T09:11:31.637Z,
  "id": 15,
  "name": "Bui Hi",
}
`;
